src/intentManager.js

```diff
@@ -108,8 +116,8 @@ define([
         writeInnerTagXML(xmlWriter, 'extra', properties.androidIntentExtra);
         if (properties.docTemplate) {
             writeInnerTagXML(xmlWriter, 'extra', {
-                'cc:print_template_reference': properties.docTemplate
-            });
+                'cc:print_template_reference': '.'
```

Comment:
The '.' might have to change to './text()' or something else, depending on how @ctsims weighs in.

Right now this will create the tag:
```
<extra key="cc:print_template_reference" ref=".">jr://file/commcare/text/question1.html</extra>
```
On the mobile side we were having trouble resolving/contextualizing the '.' xpath expression. Currently we have a hack to grab the node's text, if the key was 'cc:print...', instead of contextualizing the ref. This is unsustainable and we should figure out the correct xpath expression to put here such that the node's value is used.

I'm going to take a look at why the contextualization of '.' is causing issues.